internals-js/src/definitions.ts

```diff
@@ -1402,6 +1406,51 @@ export class Schema {
   specifiedByDirective(schema: Schema): DirectiveDefinition<{url: string}> {
     return this.getBuiltInDirective(schema, 'specifiedBy');
   }
+
+  elementByCoordinate(coordinate: string): NamedSchemaElement<any, any, any> | undefined {
```

Comment:
Linked to https://github.com/graphql/graphql-spec/pull/794. Not sure there is better place to link yet since it's one of those graphQL spec changes that seems pretty much finalised but not quite published yet.